test/scenario/metering-plans/src/test/test.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+const moment = require('abacus-moment');
+const oauth = require('abacus-oauth');
+const request = require('abacus-request');
+const util = require('util');
+const uuid = require('uuid');
+
+const doGet = util.promisify(request.get);
+const doPost = util.promisify(request.post);
+
+const env = {
+  secured: process.env.SECURED === 'true',
+  systemClientId: process.env.SYSTEM_CLIENT_ID,
+  systemClientSecret: process.env.SYSTEM_CLIENT_SECRET,
+  authServerURL: process.env.AUTH_SERVER || 'http://localhost:9882',
+  collectorURL: process.env.COLLECTOR_URL || 'http://localhost:9080',
+  reportingURL: process.env.REPORTING_URL || 'http://localhost:9088',
+  provisioningURL: process.env.PROVISIONING_URL || 'http://localhost:9880',
+  pollInterval: process.env.POLL_INTERVAL || 300
+};
+
+describe('standard services hours scenario test', () => {
+  const timestamp = moment.now();
+  const hourInMillis = 60 * 60 * 1000;
+  const resourceId = 'sampler-test-resource-id';
+  const meteringPlanId = 'standard-services-hours';
+  const planName = 'standard';
+
+  let usageDoc;
```

Comment:
This variable should not be global. The only place you are using it is in the `it` below.